### PR TITLE
adding CSR approval requirement to AWS

### PR DESCRIPTION
**Pull request by kalexand-rh**
@wking, PTAL?

### CODE REVIEW

File: modules/installation-aws-user-infra-requirements.adoc [hunk 1]
```diff
@@ -156,6 +156,19 @@ If `m4` instance types are not available in your region, such as with
 
 You might be able to use other instance types that meet the specifications of these instance types.
 
+[id="csr-management-aws_{context}"]
+== Certificate signing requests management
+
+Because your cluster has limited access to automatic machine management when you
+use infrastructure that you provision, you must provide a mechanism for approving
+cluster certificate signing requests (CSRs) after installation. The
+`kube-controller-manager` only approves the kubelet client CSRs. The
```
Comment (rphillips) [inline modules/installation-aws-user-infra-requirements.adoc]: ```The`kube-controller-manager` only approves the kubelet client CSRs after the initial node registration.```

File: modules/installation-aws-user-infra-requirements.adoc [hunk 2]
```diff
@@ -156,6 +156,19 @@ If `m4` instance types are not available in your region, such as with
...
+`machine-approver` cannot guarantee the validity of a serving certificate
+that is requested by using kubelet credentials because it cannot confirm that
+the correct machine issued the request. You must determine and implement a
+method of verifying the validity of the kubelet serving certificate requests
```
Comment (rphillips) [inline modules/installation-aws-user-infra-requirements.adoc]: You must determine and implement a method of verifying the Node's validity, and either approve or deny the certificate signing request(s) requested by the infra node(s).

**Comment by rphillips** [comment]
This also applies to nodes not backed by a Machine. That would include Metal, BYO-RHEL, and other installs that do not use AWS, GCP, Openstack, or Azure as a platform.

**Comment by rphillips** [comment]
Since we are adding this blurb to other sections, I'll retract my comments on changing the wording.

/lgtm

**Comment by kalexand-rh** [comment]
@jianlinliu, based on some discussions on slack, I'd like to add the note from https://bugzilla.redhat.com/show_bug.cgi?id=1720178 to the assemblies for AWS UPI, GCP UPI, and BYO RHEL nodes. (The fix for the bug addressed this issue for vSphere and bare metal.) Do you agree with this change?

**Comment by jianlinliu** [comment]
LGTM.

**Comment by kalexand-rh** [comment]
Thank you! Merging.

**Pull request merged**